script.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-plusplus */
+const $books = document.querySelector('.books');
+const $title = document.querySelector('.title');
+const $author = document.querySelector('.author');
+const $addButton = document.querySelector('.addButton');
+
+document.addEventListener('DOMContentLoaded', () => {
+  class Books {
+    constructor() {
+      this.bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+      this.arr = [];
+      this.bookList.forEach((book) => {
+        $books.insertAdjacentHTML(
+          'beforeend',
+          `
+            <span><p>${book.title}<br>${book.author}</p><button type="button" id="${book.title}${book.author}">Remove</button><hr></span>
+          `,
+        );
+        this.arr.push(`${book.title}${book.author}`);
+        const $removeButton = document.getElementById(`${book.title}${book.author}`);
+        $removeButton.addEventListener('click', () => {
+          $removeButton.parentElement.remove();
+          const arrayObjects = JSON.parse(localStorage.getItem('bookList'));
+          for (let i = 0; i < arrayObjects.length; i++) {
+            if (
+              `${arrayObjects[i].title}${arrayObjects[i].author}`
+              === $removeButton.getAttribute('id')
+            ) {
+              arrayObjects.splice(i, 1);
+              break;
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228684176-072472bf-c467-4d41-a92a-0c0d9a8f7593.png)


- [ ] When we remove something then it is removed from your localStorage but I guess you have a collection of books with the name this.booklist which is not being affected when you remove any todo which is totally against the project requirements :fearful: Kindly utilize that collection this.booklist to follow the project requirements :+1:
- [ ] As required in this project, It would be nice if you make a function to remove your book because I can not see any remove function because you are doing it in a class which is against the project requirement :fearful:

Reply:
Done @HAMMAS-SALEEM